utils/to_string.hh

```diff
@@ -9,16 +9,14 @@
 #pragma once
 
 #include <seastar/core/sstring.hh>
-#include <vector>
-#include <sstream>
-#include <unordered_set>
-#include <set>
-#include <optional>
-#include <list>
-#include <map>
+#include <string>
 
 #include "seastarx.hh"
-#include "utils/chunked_vector.hh"
+
+#include "bytes.hh"
+
+#include <boost/test/utils/basic_cstring/basic_cstring_fwd.hpp>
```

Comment:
I don't think we can - or should - use boost/test includes anywhere outside tests. Please don't do it.